src/test/java/org/opensearch/security/dlic/rest/api/AuditApiActionTest.java

```diff
@@ -69,8 +71,8 @@ public AuditApiActionTest(String configEndpoint, String endpoint){
     @Parameterized.Parameters
     public static Iterable<String[]> endpoints() {
         return Arrays.asList(new String[][] {
-                {"/_opendistro/_security/api/audit/config", "/_opendistro/_security/api/audit"},
-                {"/_plugins/_security/api/audit/config", "/_plugins/_security/api/audit"}
+                {LEGACY_OPENDISTRO_PREFIX + "/api/audit/config", LEGACY_OPENDISTRO_PREFIX + "/api/audit"},
```

Comment:
`CONFIG_ENDPOINT` is `ENDPOINT +"/config"`, is it necessary to have `CONFIG_ENDPOINT` defined? Please also check other places for the same pattern.

Reply:
Done